tests/basic/kubevirt_test.go

```diff
@@ -45,3 +47,58 @@ var _ = Describe("{AddNewDiskToKubevirtVM}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+var _ = Describe("{KubeVirtLiveMigration}", func() {
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("KubeVirtLiveMigration", "Live migrate a kubevirtVM", nil, 0)
+	})
+
+	var contexts []*scheduler.Context
```

Comment:
nit: contexts -> appCtxs for readability

Reply:
Done